snippets/coffee-ember-object

```diff
@@ -0,0 +1,243 @@
+
+'.source.coffee':
```

Comment:
Please move merge this file's content with existing snippets based on it's type (function, module, property)